test/verify/check-users

```diff
@@ -697,6 +698,7 @@ class TestAccounts(MachineCase):
         # Header + one line of logins
         b.wait_js_func("ph_count_check", "#account-logs tr", 2)
 
+    @todoPybridge(flaky=True)
```

Comment:
How does it fail? It [succeeded](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230124-181926-0f2cd015-fedora-37-pybridge/log.html#335-2) before. `flaky=true` is really awkward as nothing will tell us when it starts to work, and also it doesn't tell us if we accidentally break it harder.

TBH this really shouldn't be blamed so easily on the pybridge. This page only uses rather simple cockpit.file() and cockpit.spawn(), which have worked fine for a long time already.